### ISSUE TITLE
Do not drag notes with forward/back mouse buttons

### ISSUE THREAD

**Issue by rualark** [paulrosen/abcjs#362]
When I click forward mouse button to go to previous page and move a mouse slightly unintentionally, the following unfortunate chain of events happen, which leads to notation corruption:

1. abcjs drags note
2. visual editor records changed note to browser local storage
3. browser goes to previous page

At this point in time, notation is unintentionally corrupted and there is no way to undo, because we already left the page.

**Comment by paulrosen** [comment]
Can you put a breakpoint in selection.js in the function `function mouseDown(ev) {` and let me know what the object looks like? I'm not sure what events happen for the "forward mouse button" but there must be a modifier flag on it.

**Comment by rualark** [comment]
How can I put a breakpoint if I use abcjs like this?
```
<script src="plugin/abcjs-5.12.0/abcjs_basic_5.12.0.js" type="text/javascript"></script>
```

**Comment by paulrosen** [comment]
As long as you are using the version with "-min" in the name, you should be able to find the line and add the breakpoint.

**Comment by rualark** [comment]
Thanks, I managed to do that (with version without "-min"):

https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button

It seems that dragging should occur only when main mouse button is pressed (button=0)

https://i.imgur.com/Ona9TP3.png

**Comment by paulrosen** [comment]
fixed in 6.0.0-beta.13

**Comment by rualark** [comment]
✅  Tested successfully
